README.md

```diff
@@ -1,13 +1,15 @@
 # Release Information
 
-- **Version**:  2.1.0
+- **Version**:  3.0.0
 - **Certified**: Yes
 - **Publisher**: Fortinet
 - **Compatible Version**: FortiSOAR v7.6.1 and later
 - [Release Notes](./release_notes.md)
```

Comment:
Release notes missing

Reply:
It was work in progress. The release notes have now been added to the PR